junit-vintage-engine/src/main/java/org/junit/vintage/engine/discovery/ClassSelectorResolver.java

```diff
@@ -59,7 +59,7 @@ public Resolution resolve(UniqueIdSelector selector, Context context) {
 	}
 
 	private Resolution resolveTestClass(Class<?> testClass, Context context) {
-		if (!classFilter.test(testClass)) {
+		if (!classFilter.match(testClass) || !classFilter.match(testClass.getName())) {
```

Comment:
@marcphilipp I think you wanted to do something different here but I don’t remember exactly what it was

Reply:
@juliette-derancourt Is it okay if I push directly to your branch?

Reply:
I assumed it'd be okay and pushed a commit. :slightly_smiling_face:

Reply:
@marcphilipp thank you!